gnovm/cmd/gno/test.go

```diff
@@ -133,9 +134,13 @@ func execTest(cfg *testCfg, args []string, io *commands.IO) error {
 		cfg.rootDir = guessRootDir()
 	}
 
-	pkgPaths, err := gnoPackagesFromArgs(args)
+	paths, err := gnoPackagesFromArgs(args)
```

Comment:
better to leave it as pkgPaths so as to differentiate from file paths.

Reply:
but they can be file paths too.

e.g. `paths` can contain `/path/to/pkg`, `/path/to/pkg/file_test.gno`, or `/path/to/pkg/z[N]_filetest.gno`

---
Later I'm also planning to support patterns in args. e.g. `gno test ./...`, `gno test /examples/.../pkg` etc

`paths` can be pkgpath. filepath or both depends on the `args`

Reply:
Let's do the renaming in a new PR so we can focus on names.

---

Suggestion:

```suggestion
	targets, err := targetsFromPatterns(args)
```

```go
// targetsFromPatterns returns a list of local dirs or single files
// intended to be used by gno commands such as `gno test`.
func targetsFromPatterns(args []string) []target {
	// TODO: support './...'
}
```